git-branchless-record/src/lib.rs

```diff
@@ -150,15 +150,15 @@ fn record(
                 &repo,
                 &snapshot,
                 event_tx_id,
-                message.as_deref(),
+                messages,
             )?);
         }
     } else {
         let args = {
             let mut args = vec!["commit"];
-            if let Some(message) = &message {
-                args.extend(["--message", message]);
-            }
+            messages
+                .iter()
+                .for_each(|message| args.extend(["--message", message]));
```

Comment:
I think we can write it something like this:

```suggestion
            messages.extend(messages.iter().flat_map(|message| ["--message", message]));
```

Reply:
Oh, of course. Thanks for spotting that! Fixed.